pkg/apis/certmanager/v1alpha3/types_issuer.go

```diff
@@ -192,6 +192,9 @@ type VaultIssuer struct {
 	// "my_pki_mount/sign/my-role-name".
 	Path string `json:"path"`
 
+	// Namespace is the name of the vault namespace
+	Namespace string `json:"namespace,omitempty"`
```

Comment:
Should also have the `// +optional` marker 🙂

Reply:
Thanks @munnerz, fixed this.  Also I have added `namespace` in all api-versions (v1alpha3, v1alpha2, v1beta1) of VaultIssuer. Let me know if this is the correct way.